app/controllers/WebchatController.scala

```diff
@@ -22,7 +22,7 @@ import javax.inject.{Inject, Singleton}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, RequestHeader}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import views.html.webchat._
-
+import views.html.CUIViews.AskHmrcOnlineCUIView
```

Comment:
Same with this one. Is it still needed?

Reply:
Hi Lorenzo, Yes this is no more required, hence i am removing it.